lldb/source/Commands/Options.td

```diff
@@ -1048,6 +1048,9 @@ let Command = "thread backtrace" in {
   Arg<"FrameIndex">, Desc<"Frame in which to start the backtrace">;
   def thread_backtrace_extended : Option<"extended", "e">, Group<1>,
   Arg<"Boolean">, Desc<"Show the extended backtrace, if available">;
+  def thread_backtrace_full : Option<"filtered", "f">, Group<1>,
+  Arg<"Boolean">,
```

Comment:
```suggestion
```
If you make this a `Boolean` you have to pass an argument to the option (`thread backtrace -f true`). If you don't specify it, you don't have to parse the argument into a boolean `SetOptionValue`, you just have to check the short option and set your flag.

Reply:
The problem is that this is True by default. I guess I could negate the option? What's a good name `-u`, `--unfiltered`?

Reply:
suggestion: `-a`, `--all-frames` (like the option for `ls`) and in the help for that option we could mention that it shows all the frames including the ones hidden by a recognizer.